Procfile

```diff
@@ -1,3 +1,3 @@
-web: app/app.py --config=app/config.py
-db: mongod --dbpath data/db
+web: python app/app.py --config=app/config.py
+db: mongod --dbpath c:\mongodb\data
```

Comment:
Thinking about it more, I think I agree I should just document it in the README, since as is it won't work both ways anyhow due to different pathings/need for elevated permissions, etc.